solution.cpp

```diff
@@ -1,23 +1,121 @@
-/* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+//* ----------------------------------------------------------------------------
+* Copyright &copy; 2015 George Calderon <gcald@csu.fullerton.edu>
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)
+* ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
- */
+* This program converts integers into Roman numerals.
+*/
 
 // TODO: `#include`s and function prototypes
 
+#include <iostream>
+#include <string>
+using std::cout;
+using std::cin;
+using std::string;
+using std::endl;
+
 // ----------------------------------------------------------------------------
+string ConRom(int a);
+string again, roman;
 
 int main() {
-    // TODO
-
-    return 0;  // success
+	// TODO
+	while (again != "n" && again != "N") // Checks user input to loop
```

Comment:
`again` isn't initialized at this point, so the program's behavior is undefined (though, there's a good chance it will *usually* work -- but those are often the hardest kind of bugs to track down) -- gotta be careful with that!